lib/ansible/modules/network/nxos/nxos_vrf_af.py

```diff
@@ -70,7 +70,6 @@
 - nxos_vrf_af:
     vrf: ntc
     afi: ipv4
-    safi: unicast
```

Comment:
Remove this from `DOCUMENTATION` above, too? Or at least mention its uselessness and pending removal?